src/styles/nomalize.css

```diff
@@ -0,0 +1,20 @@
+* {
+  margin: 0;
+}
```

Comment:
Don't reset margins for elements with `*` selector, it can cause unexpected behaviour